.github/workflows/deploy.yaml

```
@@ -0,0 +1,66 @@
+
+on:
+  push:
+    branches:
+      - "**" # This will trigger the workflow on push to any branch
+
+jobs:
+  deploy:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout code
+        uses: actions/checkout@v2
```

actionlint (1.7.4)

the runner of "actions/checkout@v2" action is too old to run on GitHub Actions. update the action's version to fix this issue

(action)